cloudinit/cmd/devel/render.py

```diff
@@ -63,7 +63,10 @@ def handle_args(name, args):
     if args.instance_data:
         instance_data_fn = args.instance_data
     else:
-        paths = read_cfg_paths()
+        try:
+            paths = read_cfg_paths()
+        except (IOError, OSError):
+            return 1
```

Comment:
Since init.read_cfg() no longer raises an OSError/IOError (and instead logs warnings) I don't think we need this try/except here anymore right?

Reply:
Removed.